README.md

```diff
@@ -15,6 +15,28 @@ Features:
 See info below, the [API docs](http://godoc.org/github.com/cretz/bine), and the [examples](examples). The project is
 MIT licensed. The Tor docs/specs and https://github.com/yawning/bulb were great helps when building this.
 
+## Installing
+Bine requires tor-static, which cannot be installed by go's package manager.
```

Comment:
Bine does not require tor-static. That's only if you want to link it statically (i.e. you opt in by importing `github.com/cretz/bine/process/embedded`). This is mentioned under "Example".

Reply:
I can't get it to work without tor-static existing, so I guess the example doesn't work.